rethinkdb/README.md

```diff
@@ -0,0 +1,32 @@
+# Rethinkdb Integration
+
+## Overview
+
+Get metrics from rethinkdb service in real time to:
+
+* Visualize and monitor rethinkdb states
+* Be notified about rethinkdb failovers and events.
+
+## Installation
+
+Install the `dd-check-rethinkdb` package manually or with your favorite configuration manager
+
+## Configuration
+
+Edit the `rethinkdb.yaml` file to point to your server and port, set the masters to monitor
```

Comment:
Should probably add ' user and password if auth enabled' and perhaps a reference to https://rethinkdb.com/api/javascript/grant/ with a suggested user (like datadog)